server/app/middlewares/orders.js

```diff
@@ -0,0 +1,109 @@
+
+import {
+  allOrders,
+  getOne,
+  placeNewOrder,
+  updateOrder,
+  allfood,
+} from './../controllers/orders';
+
+/** Get all Orders
+ * @param {strings}
+ * @return {object}
+ * @public
+*/
+const getAllOrders = (req, res) => {
+  allOrders().then((orders) => {
+    res.send({
+      status: 'success',
+      message: 'orders were fetched succcessfully',
+      data: orders,
+    });
+  }).catch(() => res.status(404).send({
```

Comment:
Since you are returning error messages from the Promise rejections, why not send those back to the user?

Reply:
done